src/views/Teams/team.test.jsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
The file name should be Team.test.jsx. It's important the test file names match the file they are testing.